tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -595,14 +598,20 @@ def create_pool(self, **kw):
             self.exec_cmd(cmd="rbd pool init {}".format(kw.get("poolname")))
 
     def create_image(self, **kw):
-        cmd = "rbd create -s {} {} --image-feature exclusive-lock,journaling".format(
-            kw.get("size", "2G"), kw.get("imagespec")
+        cmd = "rbd create {} --size {}".format(
+            kw.get("imagespec"), kw.get("size", "2G")
         )
-
         if kw.get("datapool", self.datapool):
             cmd = cmd + " --data-pool {}".format(kw.get("datapool", self.datapool))
         self.exec_cmd(cmd=cmd)
 
+    # Enable the required image feature on image
+    def image_feature(self, **kw):
+        cmd = "rbd feature enable {} {}".format(
```

Comment:
Could you please add doc string and example for this method

Reply:
Done.